Framework/Core/src/DeviceSpecHelpers.cxx

```diff
@@ -92,7 +92,8 @@ struct ExpirationHandlerHelpers {
       auto* timer = (uv_timer_t*)(malloc(sizeof(uv_timer_t)));
       timer->data = &state;
       uv_timer_init(state.loop, timer);
-      uv_timer_start(timer, detail::timer_callback, period / 1000, period / 1000);
+      constexpr size_t resolution = 100;
+      uv_timer_start(timer, detail::timer_callback, period / 1000 / resolution, period / 1000 / resolution);
```

Comment:
I am not sure about this. Do you mean that you want to fire a timer 100 times per second in order to satisfy a 1 second timer? Maybe resolution should be a function of the period / smaller number, no? A lapping timer does not come for free.

Reply:
maybe you could fire it at 90% of the interval the first time (or whatever you think the the jitter will be) and then some smaller intervals.